### DISCUSSION TITLE
CTkTextbox get value

### DISCUSSION THREAD

**Discussion by DyNgPinasOfficial** [TomSchimansky/CustomTkinter#383] [Q&A]
- CTkTextbox is not yet available on the Wiki of this repo and I'm still finding a way to get the input value from it.
- I tried the methods from Tkinter Textbox but still do not work.
- I also tried to use the method of CTkEntry of getting its value but it doesn't work with CTkTextbox.

How can I get the input value? or it still not implemented on the current version of Custom Tkinter?

**Comment by Barry-sze** [comment]
I have the some problem.
